subprojects/README.md

```diff
@@ -0,0 +1,28 @@
+# Subprojects used by hirte
+
+This directory contains all external subprojects used by hirte as git submodules. These are integrated into the main
+project using mesons [subproject feature](https://mesonbuild.com/Subprojects.html).
+
+Currently, hirte uses the following external projects:
+
+- [hashmap.c](https://github.com/engelmi/hashmap.c.git)
+- [inih](https://github.com/benhoyt/inih.git)
+
+Unfortunately, `hashmap.c` doesn't provide a `meson.build` file and patching on the fly would result in uncommitted
+changes. Because of that a fork has been created containing the necessary `meson.build` and is used for now.
+
+## Adding submodules
```

Comment:
Not sure we need this section. But, if we have it, I think there's also the step of configuring the `meson.build` file, no?

Reply:
Good point, will update the readme!